src/io/flutter/run/daemon/FlutterDevice.java

```diff
@@ -44,6 +59,20 @@ public boolean emulator() {
     return myEmulator;
   }
 
+  @Nullable
+  public String category() {
+    return myCategory;
+  }
+
+  @Nullable
+  public String platformType() {
+    return myPlatformType;
+  }
+
+  public boolean ephemeral() {
```

Comment:
just a question: what does ephemeral mean in the context of devices?

Reply:
I'll add some javadoc; ephemeral here means a device that isn't persistent on the machine (Chrome, or the 'MacOs' device are persistent; an iPhone would be an ephemeral device).